modules/aggregate/spec/spec_helper.rb

```diff
@@ -2,6 +2,9 @@
 
 require_relative '../../../vendored/require_vendored.rb'
 
+# TODO: What is a better way to add this
+$LOAD_PATH.unshift(File.join(File.dirname(__FILE__), '../../../spec/lib'))
```

Comment:
Something like https://github.com/puppetlabs/bolt/blob/master/Rakefile#L60 might be slightly better.